test/unit/lib/index.test.ts

```diff
@@ -52,4 +52,28 @@ describe('buildDepGraph', () => {
       `Unable to find dependencies in poetry.lock for package: ${missingPackage}`,
     );
   });
+
+  it('should resolve any dependencies defined with an underscore to hash', () => {
```

Comment:
dash?

Reply:
@razzmadaz looks like I amalgamated dash and hyphen 😅 good spot.